src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/client/ApiClientServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.client;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.sdk.ApiClientService;
+import uk.gov.companieshouse.sdk.manager.ApiSdkManager;
+
+import java.io.IOException;
+
+@Component
+public class ApiClientServiceImpl implements ApiClientService {
+
+
+    @Override
+    public ApiClient getApiClient() {
+        return ApiSdkManager.getSDK();
```

Comment:
You could add a test for these, just check that you get a class of the correct type back and not null?

Reply:
I have taken a look, I'd have a lot of trouble mocking this, the libraries it calls use a lot of resources.

Reply:
you don't have to mock it, just call getApiClient() and see if you get an ApiClient back.

Reply:
It would make no sense mocking it as it's the only line in this method. If you don't mock it, in a test, does the call return an object or fall over?

(In the ROE API we didn't unit test this class, but that doesn't necessarily mean we shouldn't add a useful test here, if achievable)

Reply:
I tried to just call the function but the test fails because it requires environment variables referred to within the library, so it does look like we need to mock something here we cannot just call it.